iphone/Maps/UI/PlacePage/Components/PlacePageHeader/PlacePageHeaderViewController.swift

```diff
@@ -50,7 +50,35 @@ extension PlacePageHeaderViewController: PlacePageHeaderViewProtocol {
     }
   }
 
-  func setTitle(_ title: String) {
-    titleLabel?.text = title
+  func setTitle(_ title: String?, secondaryTitle: String?) {
+    guard let title = title,
+          let titleFont = StyleManager.shared.theme?.fonts.medium20 else {
+      titleLabel?.attributedText = nil
+      return
+    }
+
+    let titleAttributes: [NSAttributedString.Key: Any] = [
+      .font: titleFont,
+      .foregroundColor: UIColor.blackPrimaryText()
+    ]
+
+    let attributedText = NSMutableAttributedString(string: title, attributes: titleAttributes)
+
+    guard let secondaryTitle = secondaryTitle,
+          let secondaryTitleFont = StyleManager.shared.theme?.fonts.medium16 else {
+      titleLabel?.attributedText = attributedText
+      return
+    }
+
+    let paragraphStyle = NSMutableParagraphStyle()
+    paragraphStyle.paragraphSpacingBefore = 8
```

Comment:
How did you choose 8 here? Why 8 and not, say, 0? Does it increase the vertical indent from the top of the Place Page to the first line of the title?

Reply:
In this case It's spacing between title and secondaryTitle. Picked "by eye".
0:
<img width="393" alt="Screenshot 2022-11-07 at 00 54 22" src="https://user-images.githubusercontent.com/11923691/200187134-05a2b235-0f39-419d-81d4-b2326d2bf106.png">
4:
<img width="393" alt="Screenshot 2022-11-07 at 00 59 01" src="https://user-images.githubusercontent.com/11923691/200187126-2f556109-dd9e-44f7-8517-064a53b22b25.png">
8:
<img width="391" alt="Screenshot 2022-11-07 at 00 55 04" src="https://user-images.githubusercontent.com/11923691/200187130-1634efd3-e977-4542-96e1-e118767be7f8.png">

Reply:
What about 2? It probably should be the best, as the titles are related.

Reply:
With 2 points:

<img width="390" alt="Screenshot 2022-11-11 at 22 23 22" src="https://user-images.githubusercontent.com/11923691/201371112-145fd4d4-eb38-4824-b6a2-7722498c8d5b.png">